src/util/navigation.js

```diff
@@ -185,3 +185,9 @@ export const origin = () =>
 export function isPublicPath (path) {
   return (path.startsWith('/public'))
 }
+
+export function findCommentId (url) {
```

Comment:
yeah seems like we should usually (always?) use the routeParam instead of a manual thing like this. unless we are using the #